homeassistant/components/unmanic/entity.py

```diff
@@ -0,0 +1,33 @@
+"""Unmanic entity class."""
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, NAME
+
+
+class UnmanicEntity(CoordinatorEntity):
+    """Class to represent an Unmanic entity."""
+
+    def __init__(self, coordinator, config_entry, device_id):
+        """Initialize."""
+        self.config_entry = config_entry
+        self._device_id = device_id
+        super().__init__(coordinator)
+
+    @property
+    def device_info(self):
+        """Unmanic device information."""
+        if self._device_id is None:
```

Comment:
When can this happen? Does this mean that the entity is not set up?

Reply:
Yes, this used to mean it in an older iteration of the integration I was working on. It no longer seems necessary and I've removed it